features/suites/Auth/login.feature

```diff
@@ -13,5 +13,29 @@ Feature: Login into the application
     Then I should be on "/home" page
     And I should see "Welcome"
 
-  Scenario: As a developer I wanna create a merge issue
```

Comment:
What's that? Why did you remove the merge issue? I liked that!

Reply:
i don't accept because i don use phpstorm

Reply:
Please send me a phpstorm license, I send nud3s.